src/instance.ts

```diff
@@ -17,14 +17,52 @@
 'use strict';
 
 import * as arrify from 'arrify';
-const common = require('@google-cloud/common-grpc');
-import {ServiceObjectConfig} from '@google-cloud/common';
-import {paginator} from '@google-cloud/paginator';
-import {promisifyAll} from '@google-cloud/promisify';
 import * as extend from 'extend';
-import * as is from 'is';
+const common = require('@google-cloud/common-grpc');
 import snakeCase = require('lodash.snakecase');
-import {Database} from './database';
+import { ServiceObjectConfig, Metadata, GetConfig } from '@google-cloud/common';
+import { paginator } from '@google-cloud/paginator';
+import { promisifyAll } from '@google-cloud/promisify';
+import { Operation as GaxOperation } from 'google-gax/build/src/longrunning';
+import { ServiceError } from 'grpc';
+
+import { google as instance_admin_client } from '../proto/spanner_instance_admin';
+import { google as database_admin_client } from '../proto/spanner_database_admin';
+
+import { Database } from './database';
+import { SessionPoolOptions } from './session-pool';
+import { Any } from './common';
+import { Spanner } from '.';
+
+export interface CreateDatabaseCallback {
+  (err?: Error | null,
+    database?: Database | null,
+    operation?: instance_admin_client.longrunning.Operation | null,
+    resp?: database_admin_client.spanner.admin.database.v1.Database): void;
+}
+export interface CreateDatabaseRequest extends database_admin_client.spanner.admin.database.v1.ICreateDatabaseRequest {
+  poolOptions?: SessionPoolOptions;
+  poolCtor?: {};
+  schema?: string;
+}
+export interface ExistsCallback {
+  (err?: Error | null, exists?: boolean | null): void;
+}
+export interface GetDatabasesCallback {
+  (err: Error | null | undefined, databases: Database[], apiResponse: database_admin_client.spanner.admin.database.v1.Database[]): void;
+}
+export interface GetInstanceCallback {
+  (err?: Error | null, operation?: Instance, metadata?: Metadata): void;
```

Comment:
as per this [line](https://github.com/joecervino/nodejs-spanner/blob/instance-noImplicitAny/src/instance.ts#L490) I believe the signature should be 
```suggestion
  (err?: Error | null, instance?: Instance, metadata?: instance_admin_client.spanner.admin.instance.v1.IInstance): void;
```